script/create_wheel_wrapper.sh

```diff
@@ -1,61 +1,57 @@
- #!/bin/bash -e
+#!/bin/bash -e
 
 PYTHON_VERSION=$1
-
 BUILD_SCRIPT_PATH=${2:-""}
+EXTRA_ARGS="${@:3}" # Capture all additional arguments passed to the script
+CURRENT_DIR="${PWD}"
 
+# Temporary build script path
 if [ -n "$BUILD_SCRIPT_PATH" ]; then
     TEMP_BUILD_SCRIPT_PATH="temp_build_script.sh"
 else
     TEMP_BUILD_SCRIPT_PATH=""
 fi
 
-EXTRA_ARGS="${@:3}" # Capture all additional arguments passed to the script
-
-CURRENT_DIR="${PWD}"
-
-# install required dependencies
-echo "Installing dependencies required for python installation..."
-yum install -y sudo zlib-devel wget ncurses git gcc gcc-c++ make cmake
-yum install -y libffi libffi-devel sqlite sqlite-devel sqlite-libs openssl-devel
-
 # Function to install a specific Python version
 install_python_version() {
     local version=$1
+    echo "Installing Python version: $version"
     case $version in
     "3.9" | "3.11" | "3.12")
         yum install -y python${version} python${version}-devel python${version}-pip
         ;;
     "3.10")
         if ! python3.10 --version &>/dev/null; then
-            echo "Now installing python3.10..."
-            wget https://www.python.org/ftp/python/3.10.14/Python-3.10.14.tgz
-            tar xf Python-3.10.14.tgz
-            cd Python-3.10.14
-            echo "Building python3.10..."
+            echo "Installing dependencies required for python installation..."
+            yum install -y sudo zlib-devel wget ncurses git gcc gcc-c++ make cmake \
```

Comment:
@nikhil-kalbande can we add complier that Amir was mentioning. If yes @Vipul-Ajmera needs to move `yum install` lines to somewhere at the top and add missing Linux distro packages from below list.
 
```
gcc
gcc-c++
gcc-gfortran
git
xz
cmake
make
yum-utils
wget
sudo
llvm
libcurl-devel
openssl
openssl-devel
libxml2-devel
bzip2-devel
libffi
libffi-devel
sqlite-devel
libpng-devel
sqlite-libs
libjpeg-devel
libicu-devel
oniguruma-devel
readline-devel
libtidy-devel
libxslt-devel
libzip-devel
diffutils
autoconf
bison-devel
git
bzip2
file
cargo
nodejs
nodejs-devel
nodejs-packaging
npm
ncurses
python3.ppc64le
python3-devel.ppc64le
zlib-devel
openblas-devel
python-clang
python3-protobuf
protobuf-c
cairo
fontconfig
yajl
tk
```

Reply:
If we are going with above, no harm to keep below things as well.
```
# Install Rust (using rustup)
RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
ENV PATH="/root/.cargo/bin:${PATH}"

# Install Go (official binaries)
RUN wget https://golang.org/dl/go1.21.0.linux-ppc64le.tar.gz && \
    tar -C /usr/local -xzf go1.21.0.linux-ppc64le.tar.gz && \
    rm go1.21.0.linux-ppc64le.tar.gz
ENV PATH="/usr/local/go/bin:${PATH}"
```

Reply:
For now we will not include in currency.. we want to keep installation specific to package